packages/backend/src/apps/formsg/__tests__/triggers/new-submission-answer-array.test.ts

```diff
@@ -0,0 +1,85 @@
+import { IExecutionStep } from '@plumber/types'
```

Comment:
eh, how come this needs to be separate and not part of the new-submission test suite?

Reply:
ok will put it together thanks!